WooCommerce/Classes/ViewRelated/Products/Variations/Edit Attributes/RenameAttributesViewModel.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import Yosemite
+
+final class RenameAttributesViewModel {
+
+    let attributeName: String
+
+    private(set) var newAttributeName: String?
+
+    init(attributeName: String) {
+        self.attributeName = attributeName
+    }
+
+}
```

Comment:
A few comments here and there, just to keep consistency with the rest of the project would be nice :-)

Reply:
Added in https://github.com/woocommerce/woocommerce-ios/pull/3768/commits/e7265f4d118e7335e751d803d29415121f675f83